GetPerfStats/index.js

```diff
@@ -0,0 +1,73 @@
+const mongodb = require('mongodb');
+
+/**
+ * Returns list of perf results on master branch, sorted by build number desc
+ * @param example - Returns results for all perf examples by default. This param can filter results for single example.
+ * @param buildLt - Returns last 50 results by default. With this param you can get 50 results before the build number specified.
+ */
+module.exports = async function (context, req) {
+    const mongodbUri = process.env["MONGODB_URI"];
+    const mongodbUser = process.env["MONGODB_USER"];
+    const mongodbPass = process.env["MONGODB_PASS"];
+
+    const mongoClient = await mongodb.MongoClient.connect(mongodbUri, { // TODO: share persistent connection among requests
+        auth: {
+            user: mongodbUser,
+            password: mongodbPass
+        }
+    });
+
+    const query = {
+        branch: 'master'
+    }
+
+    if (req.query.buildLt && /^\d+$/.exec(req.query.buildLt)) {
+        query.build = {
+            '$lt': Number(req.query.buildLt)
+        };
+    }
+
+    const project = {
+        build: 1,
+        tag: 1,
+        sha: 1
+    };
+
+    if (req.query.example && /^[\w\d]+$/.exec(req.query.example)) {
+        project[`performance.${req.query.example}`] = 1;
+    }
+    else {
+        project.performance = 1;
+    }
+
+    context.log('Query', JSON.stringify(query, null, 2));
+    context.log('Project', JSON.stringify(project, null, 2));
+
+    try {
+        const data = await mongoClient
+            .db('stardust')
+            .collection('stats')

```

Comment:
Extract to `config`?

Reply:
Done for `GetPerfStats`, I will do the same for `SaveStats` once I verify it on prod.